Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,24 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
+
+  def firefoxappdira?
+    expand = File.expand_path("~/Applications")
+    File.exist?("#{expand}/Firefox.app")
```

Comment:
Ah, I see, it's set below. Would be nice if we could perhaps be clever and output the correct location in the caveats.